### PR TITLE
Add a script that automates release prep work

### PR DESCRIPTION
**Pull request by dominiklohmann**
###  :notebook_with_decorative_cover: Description

This adds a script that automates all the release prep work. I will demonstrate its usage shortly.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

n/t

### CODE REVIEW

File: scripts/create-release [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env zsh -euo pipefail
+
+if (( $# != 1 )); then
+  >&2 echo "Error: expected new version as only argument"
+  exit 1
+fi
+
+for binary in 'gh' 'rsync' 'git' 'perl'; do
```
Comment (tobim) [inline scripts/create-release:8]:
```suggestion
for binary in 'cmake' 'gh' 'rsync' 'git' 'perl'; do
```

**Review by tobim — CHANGES_REQUESTED** [review]
Please add comments to this.

Why does it rename the changlog directory for release candidates? Imho this should only happen for the final release.

I don't understand the reason this uses rsync over `git mv`.

**Comment by dominiklohmann** [comment]
> I don't understand the reason this uses rsync over `git mv`

`git mv` does not merge directories.

> Why does it rename the changlog directory for release candidates? Imho this should only happen for the final release.

We've done it like this for rc2 as well, so if anything we can change that for the next release. I'd consider that a separate story.

**Review by tobim — APPROVED** [review]
I trust that you used that script to create the PR for rc3.

**Pull request merged**